styles.css

```diff
@@ -0,0 +1,25 @@
+body {
+    display: flex;
+    width: 100%;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
+  #img-div {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
+  #image {
+    width: 100%;
+    max-width: 80%;
+    height: auto;
+  }
+  #title {
+    text-align: center;
+  }
+  
+  #tribute-info {
+    padding: 0px 50px;
+  }
```

Comment:
there should be a new line at the end of the page here, adding linters would also help spot those simple changes.